### PR TITLE
After Login login/signup will be replaces by username  

### LINKED ISSUE #86: Ui Changes after user login
### 🐞 Describe the Bug

If a user registers and logged in the home Ui is still showing that Login/Signup buttons ,but It should be removed to something different, which will make the user aware that he/she is logged in to the wesite as a verified user !

 LGM SOC 2021
### 📹 Screenshots or Video or GIF

---

**Note:**
* Do mention if you're working on this issue under ` LGM SOC 2021 `.
* If you want to work on any issue, you should check if it has already been assigned to anyone.

### PR DESCRIPTION
**Pull request by 7saikat7**
## What do these changes do?

After User has successfully  logged in , His/Her username will be shown in the top of the page and the signup and login buttons will be replaced  which is creating mass confusion to user that they are logged in or not !

## Related Issue

Fixes #86 

## Checklist:

- [x] Have you checked there aren't other open [Pull Requests](https://github.com/Feminine-Divine/Woodog/pulls) for the same update/change?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video or GIF:

---

### CODE REVIEW

**Comment by khushishikhu** [comment]
@7saikat7 please first open the issue, get assigned on it by asking us, and then make a PR

**Comment by 7saikat7** [comment]
Ok sure !

File: Woodog/templates/partials/base.html [hunk 1]
```diff
@@ -147,10 +147,12 @@
           <a class="nav-link" href="{% url 'appdashboard' %}">Gallery</a>
         </li>
       </ul>
-
+      {% if user.is_authenticated %}
+       <a class="btn" id="login-navbtn" type="button" href="{% url 'appdashboard' %}">welcome  {{user.username}} !</a>
```
Comment (khushishikhu) [inline Woodog/templates/partials/base.html:151]:
```suggestion
       <a class="btn" id="login-navbtn" type="button" href="{% url 'appdashboard' %}">Welcome  to woodog {{user.username}} !</a>
```

**Review by khushishikhu — CHANGES_REQUESTED** [review]

**Comment by khushishikhu** [comment]
Requested some changes. Also solve the merge conflict so that I can merge this

**Review by khushishikhu — APPROVED** [review]

**Pull request merged**